dump/dump_collect.cpp

```diff
@@ -298,12 +316,11 @@ void collectDump(const uint8_t type, const uint32_t id,
             struct pdbg_target* ocmbTarget;
             pdbg_for_each_target("ocmb", target, ocmbTarget)
             {
-                if (pdbg_target_probe(ocmbTarget) != PDBG_TARGET_ENABLED)
+                if (!is_ody_ocmb_chip(ocmbTarget))
```

Comment:
Is this relevant to the current scope?

Reply:
i put it in different commit as i find this logic wrong, agree should have been different PR